sxs/waveforms/memory.py

```diff
@@ -249,13 +250,23 @@ def add_memory(h, integration_start_time=None):
     integration_start_time : float, optional
         Time at which the energy flux integral should begin.  The default is
         `h.t[0]`.
+    psi4 : WaveformModes, optional
+        WaveformModes object corresponding to psi4
 
     Returns
     -------
-    h_with_memory : WaveformModes
-        WaveformModes object corresponding to the strain with electric memory
+    (h_with_memory, psi4_with_memory) : tuple of WaveformModes
+       WaveformModes objects corresponding to the strain with electric memory
+       and psi4 with electric memory.  If psi4 is None, then psi4_with_memory
+       is also None.
```

Comment:
For backwards compatibility, we should keep the same return type (just the single waveform) when psi4 is None

```suggestion
    h_with_memory : WaveformModes
       WaveformModes object corresponding to the strain with electric memory
    psi4_with_memory : WaveformModes, optional
       WaveformModes object corresponding to `psi4` with electric memory.  If 
       `psi4` is `None`, then this is absent.
```